### ISSUE TITLE
RCS1169 False Negative 3

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#521]
RCS1169 isn't flagged on ```p```. Is this a false negative?

```csharp
namespace RCS1169_False_Negative
{
    class Class
    {
        int i;
        System.Drawing.Point p;
    }
}
```

**Comment by JosefPihrt** [comment]
First field can be made read-only (as the analyzer suggests).

Regarding the second field, RCS1169 does not apply to value types unless it is simple type (like bool) or enum.

Please see #77 for explanation.
